### ISSUE TITLE
libssh-gcrypt.so.4 missing

### ISSUE THREAD

**Issue by FakeShemp** [flathub/net.lutris.Lutris#68]
**Describe the problem**
Trying to run the itch.io version of [&lt;reality/&gt;](https://lutris.net/games/reality-2017/) crashes at start with
```
ImportError: libssh-gcrypt.so.4: cannot open shared object file: No such file or directory
```
It does start with the Lutris runtime disabled, but then has no hardware acceleration.

**Is it reproducible with native Lutris?**
No

**Steps to reproduce**
1. Install game
2. Try to run

**Expected behavior**
The game to start.

**Actual behavior**
Nothing.

**Flatpak info**
`flatpak --version`
Flatpak 1.4.3

`flatpak --gl-drivers`
nvidia-435-21
default
host

**Comment by gasinvein** [comment]
Hmm, this library is present in 64-bit Lutris runtime but not in 32-bit runtime. Is the game 32-bit?

**Comment by FakeShemp** [comment]
It's both I think. It contains executables for both and a shell script to launch with these lines inside. 

```shell
        *-x86_64|amd64)
            RENPY_PLATFORM="linux-x86_64"
            ROOT1="$ROOT"
            ROOT2="$ROOT"
                        ;;
        *-i*86)
            RENPY_PLATFORM="linux-i686"
            ROOT1="$ROOT"
            ROOT2="$ROOT"
            ;;
```

**Comment by gasinvein** [comment]
I was unable to reproduce it with the game's demo - it always starts, but shows message about software rendering. Also I didn't find any binary in the game files that is linked with `libssh-gcrypt.so.4` directly.
So, missing `libssh-gcrypt.so.4` looks like a bug with Lutris runtime.

**Comment by FakeShemp** [comment]
Hmm. I'll try it out on other computers to see if it's something local that breaks everything and report the result.

**Comment by FakeShemp** [comment]
Yup, same issue on all my machines, with the demo too. It _does_ run when I disable the Lutris runtime, so no real problem worth looking too hard into, but an issue nonetheless.

I'm thinking this is an ffmpeg problem since it uses libssh. Haven't got more guesses than that.

**Comment by gasinvein** [comment]
I'll look into the "software rendering" issue later, it seems like a completely separate one.

About `libssh-gcrypt.so.4` - since it's something in the Lutris runtime that is linked with the library, maybe report it to Lutris?

**Comment by tannisroot** [comment]
I checked and we don't actually have that library in our runtime.

**Comment by legluondunet** [comment]
obsolete issue
